kolibri/plugins/facility/assets/src/views/ManageClassPage/index.vue

```diff
@@ -92,6 +101,72 @@
         @cancel="closeModal"
         @success="handleCreateSuccess()"
       />
+
+      <ClassRenameModal
+        v-show="openRenameModal"
+        :classname="classDetails.name"
+        :classid="classDetails.id"
+        :classes="classes"
+        @cancel="closeModal"
+        @success="handleRenameSuccess()"
+      />
+      <SidePanelModal
```

Comment:
I think we should move this to be its own component. Its boating too much this ManageClassPage component.